examples/server/tests/features/steps/steps.py

```diff
@@ -685,6 +685,32 @@ def step_tokenize_set_add_special(context):
     context.tokenize_add_special = True
 
 
+@step("tokenizing with pieces")
+@async_run_until_complete
```

Comment:
You must also add these step into .feature file. Otherwise, they will not be called

Reply:
Thanks, I have added them